policy/README.md

```diff
@@ -22,6 +22,12 @@ The goal of this specification is to enable Agencies to create, revise, and publ
 
 The machine-readable format allows Providers to obtain policies and compute compliance where it can be determined entirely by data obtained internally.
 
+Policies will typically be linked to one or more associated geographies. Geography descriptions (e.g. geofences or lists of street segments) must also be maintained by the Agency indefinitely. Policies without specific geographies (global policies) are assumed to apply to the entire service area managed by the Agency. 
```

Comment:
Shouldn't mention service areas, no longer a thing. Perhaps better to say something like "assumed to apply to all jurisdictions managed by the Agency".